core-bundle/src/Resources/contao/dca/tl_module.php

```diff
@@ -289,6 +289,15 @@
 			'sql'                     => "int(10) unsigned NOT NULL default 0",
 			'relation'                => array('type'=>'hasOne', 'load'=>'lazy')
 		),
+		'overviewPage' => array
```

Comment:
maybe call this `jumpToOverview` or `jumpBack` to follow the existing `jumpTo` field?